architectures

```diff
@@ -1,8 +1,8 @@
 bashbrew-arch   variants
-amd64    jessie,jessie-slim,stretch,stretch-slim,buster,buster-slim,alpine3.9,alpine3.10
-arm32v6  jessie,jessie-slim,stretch,stretch-slim,buster,buster-slim,alpine3.9,alpine3.10
-arm32v7  jessie,jessie-slim,stretch,stretch-slim,buster,buster-slim,alpine3.9,alpine3.10
-arm64v8  jessie,jessie-slim,stretch,stretch-slim,buster,buster-slim,alpine3.9,alpine3.10
-i386     jessie,jessie-slim,stretch,stretch-slim,buster,buster-slim,alpine3.9,alpine3.10
-ppc64le  jessie,jessie-slim,stretch,stretch-slim,buster,buster-slim,alpine3.9,alpine3.10
-s390x    jessie,jessie-slim,stretch,stretch-slim,buster,buster-slim,alpine3.9,alpine3.10
+amd64    jessie,jessie-slim,stretch,stretch-slim,buster,buster-slim,alpine3.9,alpine3.10,onbuild
```

Comment:
don't think "onbuild" should be added here since it's no longer supported

Reply:
The issue is that `node:8-onbuild` still exists (and the directory still exists), so for that to continue to exist, it has to be listed here too:

```console
$ bashbrew list node | grep onbuild
node:8.16.2-onbuild
node:8.16-onbuild
node:8-onbuild
node:carbon-onbuild
```

https://github.com/docker-library/official-images/blob/f396b68d8be74ecd83dac3d6aa6f18bdb7e575d8/library/node#L21-L24

Reply:
To be clear, I'm happy to remove `onbuild`, but that does mean this final `onbuild` image goes away too, and I'm guessing it was left in on purpose?

Reply:
Maybe I'm missing where it is being used, since "onbuild" is still in the 8/architectures, but maybe this is something that the stackbrew generation needs in both places

Reply:
https://github.com/nodejs/docker-node/pull/994#discussion_r258662245

> So, this is where things got weird. What I found in testing was that the root file was somehow being used to determine the list of architecture+variant combinations that it was willing to try, so when I copied this file as-is to the root, all the jessie variants went away completely, which is why I've updated the root file here to simply list every possible variant for every supported architecture and opted instead to be explicit per-version which arch+variant combinations are supported.

Reply:
thanks! i'll see if I can figure out if that file is removable separately